demo/src/cfn/template.yaml

```diff
@@ -4,14 +4,14 @@ Transform: AWS::Serverless-2016-10-31
 
 Globals:
   Function:
-    Runtime: nodejs12.x
+    Runtime: nodejs14.x
```

Comment:
This is unrelated but I've done it too given node 12 is approaching end of life.